src/main/java/step2/RewardBoard.java

```diff
@@ -0,0 +1,31 @@
+package step2;
+
+import java.util.Arrays;
+
+public enum RewardBoard {
+    FIRST(6, 2000000000),
```

Comment:
등수 객체를 enum으로 잘 구현해주셨어요 👍 
돈과 같은 1000단위로 구분은 다음과 같이 underbar(_)를 활용하면 코드의 가독성이 좋아질 수 있어요!
```suggestion
public enum RewardBoard {
    FIRST(6, 2_000_000_000),
```